bin/main.rb

```diff
@@ -1,2 +1,29 @@
 #!/usr/bin/env ruby
-puts 'Hello World!'
+puts "\t\t\t\tHello to Tic Tac Toe game Build by Hamayun and Asad!"
+puts "\n"
+puts "Hey player-1, please enter your name:"
+ply_1 = gets.chomp
+
+puts "Hey player-2, please enter your name:"
+ply_2 = gets.chomp
+puts "\n"
+puts "#{ply_1}, you have got X"
+puts "#{ply_2}, you have got O"
+puts "\n"
+puts "****Gameboard is displayed****"
+
+puts "NOTE: The player who wants to mark the box, just write its box number."
+
+puts "#{ply_1}, you got the first turn"
+
+i = 0
+while i < 9
+    if i.even?
+        puts "#{ply_1} choose your box."
+    else
+        puts "#{ply_2} choose your box."
+    end
+    box = gets.chomp.to_i
+    puts "****Gameboard is displayed****"
+    i += 1
+end
```

Comment:
Please use a terminating variable to exit the loop instead of using numbers. Since in the next milestone you will implement the game logic, you are going to need to run the game as long as no player has won or the game is not draw. Almost all the time one player will win the game before 9 moves.

You can use something like this:

```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```